server/main.lua

```diff
@@ -1,29 +1,25 @@
 local QBCore = exports['qb-core']:GetCoreObject()
 
--- Events
-
 RegisterNetEvent('qb-recycle:server:getItem', function()
-  local src = source
-  local Player = QBCore.Functions.GetPlayer(src)
-  for _ = 1, math.random(1, Config.MaxItemsReceived), 1 do
-    local randItem = Config.ItemTable[math.random(1, #Config.ItemTable)]
-    local amount = math.random(Config.MinItemReceivedQty, Config.MaxItemReceivedQty)
-    Player.Functions.AddItem(randItem, amount)
-    TriggerClientEvent('inventory:client:ItemBox', src, QBCore.Shared.Items[randItem], 'add')
-    Wait(500)
-  end
+    local src = source
+    local player = QBCore.Functions.GetPlayer(src)
+
+    for _ = 1, math.random(1, Config.MaxItemsReceived) do
```

Comment:
I see you changed it from your first commit. `_` is used here because it is an unused variable.
If the number was used somewhere in the for loop, you would be correct!
e.g.: 
```
for i = 1, #Config.Items do
    print(Config.Items.Name, i)
end
```